.babelrc

```diff
@@ -8,7 +12,10 @@
       "presets": ["react-hmre"]
     },
     "test": {
-      "plugins": ["react-pure-components"]
+      "plugins": [
+        "react-pure-components",
+        "transform-es2015-modules-commonjs"
```

Comment:
Ah, I think this got lost in [the commit](https://github.com/buildkite/frontend/pull/282/commits/8b4c8d12a3d616f6f77a106b75c0627eca6dab36) title again, sorry:

> Jest still needs commonjs modules
> 
> This is what the docs recommend:
> 
> https://facebook.github.io/jest/docs/webpack.html#using-with-webpack-2